docs/Admin/CrossTenantMailboxMigrationValidation.md

```diff
@@ -0,0 +1,80 @@
+- You would need the Azure AD Module (https://docs.microsoft.com/en-us/powershell/azure/active-directory/install-adv2?view=azureadps-2.0#installing-the-azure-ad-module)
+- Also, you will be prompted for the SourceTenantId and TargetTenantId if you choose to run the script with the "CheckOrgs" parameter. To obtain the tenant ID of a subscription, sign in to the Microsoft 365 admin center and go to https://aad.portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Properties. Click the copy icon for the Tenant ID property to copy it to the clipboard.
+
+## PARAMETERS
+
+### CheckObjects
+This will allow you to perform the checks for the Source Mailbox and Target MailUser objects you provide. If used without the "-CSV" parameter, you will be prompted to type the identities.
+
+### CSV
+This will allow you to specify a path for a CSV file you have with a list of users that contain the "SourceUser, TargetUser" columns.
+An example of the CSV file content would be:
+
+        SourceUser, TargetUser
+        Jdoe@contoso.com, Jdoe@fabrikam.com
+        BSmith@contoso.com, BSmith@fabrikam.com
+
+### CheckOrgs
+This will allow you to perform the checks for the source and target organizations. More specifically the organization relationship on both tenants, the migration endpoint on target tenant and the existence of the AAD application needed.
+
+### SDP
+This will collect all the relevant information for troubleshooting from both tenants and be able to send it to Microsoft Support in case of needed.
+
+### LogPath
+This will allow you to specify a log path to transcript all the script execution and it's results. This parameter is mandatory.
+
+
+## EXAMPLES
+### EXAMPLE 1
+
+    .\CrossTenantMailboxMigrationValidation.ps1 -CheckObjects -LogPath C:\Temp\LogFile.txt
+
+This will prompt you to type the source mailbox identity and the target identity, will establish 2 EXO remote powershell sessions (one to the source tenant and another one to the target tenant), and will check the objects.
+
+### EXAMPLE 2
+
+    .\CrossTenantMailboxMigrationValidation.ps1 -CheckObjects -CSV C:\Temp\UsersToMigrateValidationList.CSV -LogPath C:\Temp\LogFile.txt
+
+This will establish 2 EXO remote powershell sessions (one to the source tenant and another one to the target tenant), will import the CSV file contents and will check the objects one by one.
+
+
+### EXAMPLE 3
+
+    .\CrossTenantMailboxMigrationValidation.ps1 -CheckOrgs -LogPath C:\Temp\LogFile.txt
+
+This will prompt you for the soureTenantId and TargetTenantId, establish 3 remote powershell sessions (one to the source EXO tenant, one to the target EXO tenant and another one to AAD target tenant), and will validate the migration endpoint on the target tenant, AAD applicationId on target tenant and the Orgnization relationship on both tenants.
```

Comment:
```suggestion
This will prompt you for the soureTenantId and TargetTenantId, establish 3 remote powershell sessions (one to the source EXO tenant, one to the target EXO tenant and another one to AAD target tenant), and will validate the migration endpoint on the target tenant, AAD applicationId on target tenant and the Orgnization relationship on both tenants.
```
```suggestion
This will prompt you for the sourceTenantId and TargetTenantId, establish 3 remote powershell sessions (one to the source EXO tenant, one to the target EXO tenant and another one to AAD target tenant), and will validate the migration endpoint on the target tenant, AAD applicationId on target tenant and the Organization relationship on both tenants.
```